### ISSUE TITLE
Feed Rate Slider not updating

### ISSUE THREAD

**Issue by re3Dprinting** [re3Dprinting/touchscreen#57]
Found the feed rate slider is not showing the current feedrate if feedrate is set by gcode.

**Comment by jim-thompson** [comment]
I don't believe we get feed and flow rate settings from Marlin, so this might require changing Marlin. There will also be a touchscreen change required.

**Comment by jim-thompson** [comment]
Checked the Marlin source: Marlin doesn't report feed or flow rate when set by gcode. Keeping the touchscreen in sync with the flow and feed rates in Marlin will require modifications to Marlin.
